slurp/src/main/webapp/index.html

```diff
@@ -4,18 +4,18 @@
     <meta charset="UTF-8">
     <title>Slurp</title>
     <!-- CSS -->
-    <link rel="stylesheet" href="/styles/style.css">
+    <link rel="stylesheet" href="styles/style.css">
 
     <!-- JavaScript -->
     <!-- The core Firebase JS SDK is always required and must be listed first -->
-    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
     <!-- Add any other SDKs for Firebase products that we want to use
         https://firebase.google.com/docs/web/setup#available-libraries -->
-    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-firestore.js"></script>
-    <script defer src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-firestore.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
 
-    <script defer src="/scripts/init-firebase.js"></script>
-    <script defer src="/scripts/script.js"></script>
```

Comment:
I think we want to keep the defer flags on each of these script tags per the instructions [here](https://firebase.google.com/docs/web/setup#delay-sdks-cdn).

Reply:
I think that would be the case if we didn't need it onload. These scripts load out of order, resulting in init-firebase.js not having all the prerequisites it needs to properly initialize `db`. 

I'm not 100% sure about that, though.

Reply:
Hmm, I am pretty sure that defer is what we want so that we don't load a page before the scripts page has loaded. Because we only have a few scripts it doesn't matter but I think that it is good practice based on what I read on sites such as [this one](https://javascript.info/script-async-defer#defer).

Also are they in the wrong order? Firebase script is loaded first, then Firestore, then we initialize Firebase & Firestore instances. Am I missing something here?

Reply:
I believe having defer on all our scripts is functionally the same as not having defer, with the upside that it allows the browser to continue loading the DOM. Not including defer on the scripts forces the browser to stop loading the DOM and focus instead on loading the script. Using defer both lets the script load in the background and enforces that the deferred scripts are loaded in order (unlike async which just tells the browser to greedily load).

So I think having defer on the scripts is better, since it functionally works the same as not having them with the upside that we might get a speed boost.

Reply:
I just tested it, and it seems to work alright. 
Updated.